### DISCUSSION TITLE
Solax x1 hybrid gen4 consume off power and standby mode random

### DISCUSSION THREAD

**Discussion by gazzzzton** [wills106/homeassistant-solax-modbus#712] [General]
Hi, I need your help to understand the setting "consume off power" on solax gen4, what is it used for ?
I also don't understand why at night my inverter sometimes switches from "normal mode" to "standby mode" randomly as you can see in the photo?
![Capture d'écran 2024-01-27 195306](https://github.com/wills106/homeassistant-solax-modbus/assets/63874950/46b46873-d5e0-4a0f-8a25-b575d035e1d9)

 thank you in advance :)

**Comment by yvosound** [comment]
hi
did you get an answer somewhere? 
i also wan't to know the number.solaxws_consume_off_power entity

**Comment by wiltet** [comment]
Hi,
would like to push the first question.
I've landed at the same point and could not find any explanation in the docs about "consume power off".

Maybe there is somebody reading this and can redirect it to the right person from the project?

btw. thanks a lot to everyone working on this very useful program! 


thats all I could find:
https://github.com/search?q=repo%3Awills106%2Fhomeassistant-solax-modbus+consume_off_power&type=code
